src/project/manifest/python.rs

```diff
@@ -163,9 +163,10 @@ impl PyPiRequirement {
             PyPiRequirement::Path { editable: e, .. } => {
                 *e = Some(editable);
             }
-            _ => {
+            _ if editable => {
                 tracing::error!("Cannot add editable to a non path like dependency.");
```

Comment:
Should we say `warning!("ignoring `editable` on a non path-like dependency")`
Error is very scary.